### PR TITLE
Refactoring schema conversion logic to GLTFConverter class.

### PR DESCRIPTION
**Pull request by kcoley**
- Refactors schema conversion logic to `GLTFConverter` class.
- Replaces `BuildGLTF` method with static method `Runtime.GLTFConverter.ConvertRuntimeToSchema`
- Sets the indices of the children of the `Node` class at the top level (parent node has the lower index).

### CODE REVIEW

**Comment by kcoley** [comment]
This PR also sets the children of the node at the top level (parent node has the lower index).

File: Source/Program.cs [hunk 1]
```diff
@@ -78,7 +78,8 @@ private static void Main(string[] args)
 
                     // Passes the desired properties to the runtime layer, which then coverts that data into
                     // a gltf loader object, ready to create the model
-                    wrapper.BuildGLTF(ref gltf, geometryData);
+                    Runtime.GLTFConverter.ConvertRuntimeToSchema(wrapper, ref gltf, geometryData);
+                    //wrapper.BuildGLTF(ref gltf, geometryData);
```
Comment (stevk) [inline Source/Program.cs]: Remove commented out code.

File: Source/Runtime/GLTFConverter.cs [hunk 2]
```diff
@@ -0,0 +1,1252 @@
...
+            if (specGloss.GlossinessFactor.HasValue)
+            {
+                materialPbrSpecularGlossiness.GlossinessFactor = specGloss.GlossinessFactor.Value;
+            }
+
+            return materialPbrSpecularGlossiness;
+        }
+
+        /// <summary>
+        /// Converts Runtime Quatum Rendering to Schema (Not an actual glTF feature)
```
Comment (stevk) [inline Source/Runtime/GLTFConverter.cs]: Change 'Quatum' to 'Quantum'

File: Source/Program.cs (hunk 1 above)
Comment (kcoley) [reply Source/Program.cs]: good catch

File: Source/Runtime/GLTFConverter.cs (hunk 2 above)
Comment (kcoley) [reply Source/Runtime/GLTFConverter.cs]: good catch

**Pull request merged**